model/src/main/java/com/epam/repair/model/FirstNameDTO.java

```diff
@@ -10,7 +10,7 @@
  */
 @Entity
 @Table(name = "first_name")
-public class FirstName {
+public class FirstNameDTO {
```

Comment:
Here can't be `dto` object because it's DB view

Reply:
Comment accepted. Fixed in the branch mapstruct-model.